cuda_core/tests/test_event.py

```diff
@@ -20,7 +20,13 @@ def test_event_init_disabled():
         cuda.core.experimental._event.Event()  # Ensure back door is locked.
 
 
-@pytest.mark.parametrize("enable_timing", [True, False, None])
+@pytest.mark.parametrize(
+    "enable_timing",
+    [
+        True,
+    ]
+    + ([False, None] if os.environ.get("CUDA_PYTHON_SANITIZER_RUNNING", "0") != "1" else []),
```

Comment:
Would you mind leaving a comment here?

Reply:
10d4c7a9b52512d03b1da078ae238f6e96719c2e